activerecord/test/cases/relation/where_chain_test.rb

```diff
@@ -2,17 +2,25 @@
 
 require "cases/helper"
 require "models/post"
+require "models/author"
 require "models/comment"
+require "models/categorization"
 
 module ActiveRecord
   class WhereChainTest < ActiveRecord::TestCase
-    fixtures :posts
+    fixtures :posts, :authors
 
     def setup
       super
       @name = "title"
     end
 
+    def test_missing_with_association
+      relation = Post.where.missing(:author)
+      expected = Post.left_joins(:author).where(authors: { id: nil })
+      assert_equal expected, relation
+    end
+
```

Comment:
How about a test for `.missing` with multiple associations?

Reply:
Thanks @georgeclaghorn ! I added a test for a model that has been orphaned by multiple associations. Are there other tests you can think of?